__tests__/app.test.js

```diff
@@ -117,4 +117,23 @@ describe('/api/articles', () => {
   });
 });
 
-
+describe("/api/articles/:article_id/comments", () => {
+  test("GET:200 - responds with all the comments matching the passed article id in a descending order", () => {
+    return request(app)
+      .get("/api/articles/4/comments")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.comments).toBeSortedBy("created_at", { descending: true });
+        body.comments.forEach((comment) =>{
+          expect(comment).toMatchObject({
+              comment_id: expect.any(Number),
+              body: expect.any(String),
+              article_id: 4,
+              author: expect.any(String),
+              votes: expect.any(Number),
+              created_at: expect.any(String)
+            })
+          })
+      });
+  });
+});
```

Comment:
I've check on your `main` branch and don't see any error handling tests for this end point.
What errors need to be handled? Remember, this is a parametric endpoint so you can use one your other similar endpoints for inspiration 😄